README.md

```diff
@@ -1,5 +1,14 @@
 # AMQ Multicluster Reference deployment
 
+The architecture deployed by this template is shown in this diagram:
+
+![amq-dr-reference-architecture](./media/amq-dr-arch.png)
+
+In this diagram two sites/data-centers are represented. A GTM directs traffic to either one or the other data-center. Message create by the message producer are received by the interconnect pods. These pods are configured to send the message to the brokers in the same data-center. They also have an alternative route with high-weight which sends the messages to the interconnect pods in the other data-centers, should no broker be available in the current data-center.
```

Comment:
`A GTM directs traffic to either one or the other data-center.` this is not true. the GTM is either clustered across data-center or consumed as a service.